openshift/launchpad-template.yaml

```diff
@@ -41,11 +41,13 @@ parameters:
 - description: OpenShift user name (leave empty when using Keycloak)
   displayName: OpenShift user name
   name: LAUNCHPAD_MISSIONCONTROL_OPENSHIFT_USERNAME
-  value: developer
+  value: 
+  required: true
```

Comment:
I don't think this is correct.  A) because the username and password can be left empty if we want to use a token and B) the "developer/developer" is a correct default value. Right @gastaldi ?